galsim/gsobject.py

```diff
@@ -435,7 +435,7 @@ def interpolated(self): return False
     @property
     def deinterpolated(self): return self
     @property
-    def SED(self):
+    def sed(self):
```

Comment:
I think we need a deprecated uppercase `SED` here.

Reply:
I guess so.  I don't think users ever probably duck type GSObjects as a ChromaticObject.  That's more of a convenience for us in implementation.  But it doesn't hurt to add.